src/static/css/mobile-carousel.less

```diff
@@ -0,0 +1,113 @@
+/* ==========================================================================
+   cfgov-refresh
+   Overrides for slick carousel on /the-bureau/ and /careers/working-at-cfpb/.
+   ========================================================================== */
+
+.js-mobile-carousel {
```

Comment:
You still don't want to use the `.js-` prefix for the styles, the parent block should have both `mobile-carousel` and `js-mobile-carousel` classes. `.js-` prefixes should only be used for JS hooks, not for styling. The entire idea around using the prefix is to decouple scripting from styling https://coderwall.com/p/qktuzw/decouple-javascript-classes-from-css-ones-by-using-prefix